### ISSUE TITLE
explicitly document the use of ttype for  async ops

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/saga-python#460]

**Comment by vivek-bala** [comment]
Not sure which async ops this refers to. @andre-merzky any pointers?

**Comment by andre-merzky** [comment]
All methods in SAGA can be called as async methods:

```
f = rs.filesystem.File('/etc/passwd')

# synchronous call
print f.get_size()

# this is equivalent to this version which creates a task in a final state (synchronous call)
task = f.get_size(ttype=rs.SYNC)
print task.get_result()

# this is a version creating a RUNNING task which one needs to wait on to get the result:
task = f.get_size(ttype=rs.ASYNC)
task.wait()
print task.get_result()

# this is a version creating a NEW task which one needs to run and wait on to get the result:
task = f.get_size(ttype=rs.TASK)
task.run()
task.wait()
print task.get_result()
```

This holds for *every* method of the functional API packages.  Its rarely used, but still, it needs proper documentation.

**Comment by vivek-bala** [comment]
Thanks for the info! I'll leave this ticket open then.
